test/libsolidity/util/TestFileParser.cpp

```diff
@@ -207,30 +211,37 @@ tuple<bytes, ABIType, string> TestFileParser::parseABITypeLiteral()
 	try
 	{
 		u256 number{0};
-		ABIType abiType{ABIType::None, 0};
+		ABIType abiType{ABIType::None, ABIType::AlignRight, 0};
 		string rawString;
 
 		if (accept(Token::Sub))
 		{
-			abiType = ABIType{ABIType::SignedDec, 32};
+			abiType = ABIType{ABIType::SignedDec, ABIType::AlignRight, 32};
 			expect(Token::Sub);
 			rawString += formatToken(Token::Sub);
-			string parsed = parseNumber();
+			string parsed = parseDecimalNumber();
 			rawString += parsed;
 			number = convertNumber(parsed) * -1;
 		}
 		else
 		{
-			if (accept(Token::Number))
+			if (accept(Token::HexNumber))
+			{
+				abiType = ABIType{ABIType::UnsignedDec, ABIType::AlignLeft, 32};
```

Comment:
`ABIType::UnsignedDec` is not really true anymore, is it?

Reply:
Ah, good catch. I only introduced that change in a prototype of the unpadded hex string implementation (`hex"0042"`). Will take that over and add `ABIType::Hex`.